packages/app/src/runner/events/capture-protocol.ts

```diff
@@ -0,0 +1,33 @@
+export const addCaptureProtocolListeners = (Cypress) => {
```

Comment:
```suggestion
export const addCaptureProtocolListeners = (Cypress: Cypress.Cypress) => {
```

Reply:
might have to `@ts-expect-error` the backend/runner but it should at least type the change events!

Reply:
Added types here: [1330057](https://github.com/cypress-io/cypress/pull/26508/commits/1330057398b2744d61449131913fb275cdb51415)